requirements.txt

```diff
@@ -1,6 +1,7 @@
 # Base requirements for apel
 
-MySQL-python
-python-ldap<3.4.0  # python-ldap-3.4.0 dropped support for Python 2
-iso8601
+mysqlclient==2.1.1
+python-ldap
+iso8601==1.1.0
```

Comment:
Why this version?

Reply:
Same as above. Acc. to 1.1.0 https://pypi.org/project/iso8601/1.1.0/   . It acknowledges that it works with 3.6.2+ and 3.9 which we tested the codebase and this is the latest package. Anything above that requires 3.7 codebase.